src/Pages/Channels.razor

```diff
@@ -717,7 +717,7 @@
 
     private bool OnWalletFilter(object itemValue, object searchValue)
     {
-        if (itemValue == null) return false;
+        if (itemValue == null) return true;
```

Comment:
for now ghost channels will appear with "Unknown" for which wallet opened the channel

Reply:
But, the " ghost" channels were created by NG, we have the info. I don't get this.

Reply:
If we have a request in the `ChannelOperationRequest` table, then we know the wallet that originated the funding Tx. But if the channel was originated by lncli for example, or the node had previous channels before it began to be managed by the NodeGuard, then we won't have the info about which wallet created it. But we still would want to manage those channels, right?

Reply:
Agree +1